.rubocop.yml

```diff
@@ -1,10 +1,17 @@
-inherit_from:
-  - https://shopify.github.io/ruby-style-guide/rubocop.yml
+inherit_gem:
+    rubocop-shopify: rubocop.yml
 
-LineLength:
+AllCops:
+  TargetRubyVersion: 2.4
   Exclude:
-    - test/**/*
+    - 'lib/generators/**/*'
```

Comment:
Hey @adfaris!

Sorry for coming late to the game.

Why are we excluding `'lib/generators/**/*'` here?

Reply:
Rubocop doesn't play with the template files with erb in them. https://github.com/Shopify/shopify_app/blob/f95aac7fc02412e0f6d57bcdf425cb2df1414644/lib/generators/shopify_app/add_webhook/templates/webhook_job.rb#L1

Perhaps we can be more specific here.

Reply:
Ah I see!

Yeah, maybe `'lib/generators/**/templates/*'` or similar? I haven't tried this out.

Reply:
Actually I think the real issue here is that those are not `.rb` files.

Those are `.rb.tt` files, I believe. Take a look at how rails calls them:
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/controller/templates/controller.rb.tt

/cc @tanema

Reply:
I made these changes in this PR: 

https://github.com/Shopify/shopify_app/pull/959